src/SDKs/Azure.Core/data-plane/Azure.Core/Response.cs

```diff
@@ -12,12 +12,16 @@ public abstract class Response: IDisposable
     {
         public abstract int Status { get; }
 
-        public abstract bool TryGetHeader(string name, out string value);
-
         public abstract Stream ContentStream { get; set; }
 
         public abstract string RequestId { get; set; }
 
+        public abstract bool TryGetHeader(string name, out string value);
+
+        public abstract bool TryGetHeaderValues(string name, out IEnumerable<string> values);
+
+        public abstract bool ContainsHeader(string name);
```

Comment:
>public abstract bool ContainsHeader(string name); [](start = 8, length = 49)

Is there a compelling reason to have ContainsHeader vs. just using TryGetHeader or TryGetHeaderValues to test if a header exists?

Reply:
Contains might avoid materializing header if its value is not required depending on the transport.